demo/src/screens/componentScreens/RadioButtonScreen.js

```diff
@@ -39,27 +62,51 @@ export default class RadioButtonScreen extends Component {
-                <Text marginB-20 text60 dark10>
-                  Select a color
-                </Text>
-                {this.renderRadioButton('orange', 'Orange')}
-                {this.renderRadioButton('purple', 'Purple')}
-                {this.renderRadioButton('green', 'Green')}
-                <Text marginT-10>You chose: {this.state.color}</Text>
-              </RadioGroup>
-              <RadioGroup value={this.state.textSide} onValueChange={value => this.setState({textSide: value})}>
-                <Text marginB-20 text60 dark10>
-                  Select text side
-                </Text>
-                {this.renderRadioButtonWithImageAndText('right', 'Text on right')}
-                {this.renderRadioButtonWithImageAndText('left', 'Text on left', true)}
-                <Text marginT-10>You chose: {this.state.textSide}</Text>
-              </RadioGroup>
-            </View>
+        <View flex>
+          <ScrollView style={{padding: 20}}>
+            <RadioGroup value={this.state.color} onValueChange={value => this.setState({color: value})}>
+              <Text marginB-20 text60 dark10>
+                Select a color
+              </Text>
+              {this.renderRadioButton('orange', 'Orange')}
+              {this.renderRadioButton('purple', 'Purple')}
+              {this.renderRadioButton('green', 'Green')}
+              <Text marginT-10>You chose: {this.state.color}</Text>
+            </RadioGroup>
+
+            <RadioGroup marginT-30 value={this.state.textSide} onValueChange={value => this.setState({textSide: value})}>
+              <Text marginB-20 text60 dark10>
+                Select text side
+              </Text>
+              {this.renderRadioButtonWithImageAndText('right', 'Text on right')}
+              {this.renderRadioButtonWithImageAndText('left', 'Text on left', true)}
+              <Text marginT-10>You chose: {this.state.textSide}</Text>
+            </RadioGroup>
+
+            <RadioGroup
+              value={this.state.transportationType} onValueChange={value => this.setState({transportationType: value})}
+            >
+              <Text marginV-20 text60 dark10>
+                Select transportation type (enum)
+              </Text>
+              {this.renderRadioButtonForEnum(RadioButtonScreen.transportationTypes.CAR)}
+              {this.renderRadioButtonForEnum(RadioButtonScreen.transportationTypes.WALK)}
+              {this.renderRadioButtonForEnum(RadioButtonScreen.transportationTypes.BUS)}
+              <Text marginT-10>You chose: {this.state.transportationType}</Text>
+            </RadioGroup>
+
+            <RadioGroup
+              value={this.state.messageType || null} onValueChange={value => this.setState({messageType: value})}
+            >
+              <Text marginV-20 text60 dark10>
+                Select Message type{'\n'}
+                (enum with default value)
+              </Text>
+              {this.renderRadioButton(null, 'Default')}
```

Comment:
I would include the default example with null with the first example in the screen, no need to create too many of them.